### ISSUE TITLE
Code for BigEarthNet downstream task

### ISSUE THREAD

**Issue by PlekhanovaElena** [techmn/satmae_pp#4]
Hi there,

Thank you for sharing the code and impressive results!

I was wondering how to reproduce your result on the BigEarthNet dataset?
Do you by any chance have code for it or for the dataloader?

I found the [datasets_finetune.py](https://github.com/techmn/satmae_pp/blob/main/util/datasets_finetune.py), but it seems to only load the fMoW dataset with it's own normalization etc.  

Elena

**Comment by techmn** [comment]
Hi,
Thanks for the interest in our work.
Please find the dataloader for BigEarthNet from [SeCO](https://github.com/ServiceNow/seasonal-contrast/blob/main/datasets/bigearthnet_dataset.py)

We will update the repository to include the BigEarthNet finetuning also.

**Comment by hubery1619** [comment]
Hi,
Your work is pretty interesting. Currently, the link for the dataset split txt file could not be accessed by the https as follows:
list_file = {
        'train': 'https://storage.googleapis.com/remote_sensing_representations/bigearthnet-train.txt',
        'val': 'https://storage.googleapis.com/remote_sensing_representations/bigearthnet-val.txt',
        'test': 'https://storage.googleapis.com/remote_sensing_representations/bigearthnet-test.txt'

I wonder if you could share them. Thus, I could use the repo of SeCO. Thanks so much.

**Comment by AkshitSharma1** [comment]
> Hi, Your work is pretty interesting. Currently, the link for the dataset split txt file could not be accessed by the https as follows: list_file = { 'train': 'https://storage.googleapis.com/remote_sensing_representations/bigearthnet-train.txt', 'val': 'https://storage.googleapis.com/remote_sensing_representations/bigearthnet-val.txt', 'test': 'https://storage.googleapis.com/remote_sensing_representations/bigearthnet-test.txt'
> 
> I wonder if you could share them. Thus, I could use the repo of SeCO. Thanks so much.

@techmn Yes please, that would be of great help.

**Comment by AkshitSharma1** [comment]
http://bigearth.net/downloads/BigEarthNet-v1.0.tar.gz

Seems to be no longer accessible. Please could the updated link be shared? Thanks!

**Comment by techmn** [comment]
> Hi, Your work is pretty interesting. Currently, the link for the dataset split txt file could not be accessed by the https as follows: list_file = { 'train': 'https://storage.googleapis.com/remote_sensing_representations/bigearthnet-train.txt', 'val': 'https://storage.googleapis.com/remote_sensing_representations/bigearthnet-val.txt', 'test': 'https://storage.googleapis.com/remote_sensing_representations/bigearthnet-test.txt'
> 
> I wonder if you could share them. Thus, I could use the repo of SeCO. Thanks so much.

You can find the data split for BigEarthNet [here](https://zenodo.org/records/12687186). However, I am not sure if it the same as `https://storage.googleapis.com/remote_sensing_representations/bigearthnet-test.txt`

**Comment by AkshitSharma1** [comment]
> Hi, Your work is pretty interesting. Currently, the link for the dataset split txt file could not be accessed by the https as follows: list_file = { 'train': 'https://storage.googleapis.com/remote_sensing_representations/bigearthnet-train.txt', 'val': 'https://storage.googleapis.com/remote_sensing_representations/bigearthnet-val.txt', 'test': 'https://storage.googleapis.com/remote_sensing_representations/bigearthnet-test.txt'
> 
> I wonder if you could share them. Thus, I could use the repo of SeCO. Thanks so much.


Hi @hubery1619 , did you succeed in reproducing the results for BigEarthNet? Thanks!
